### PR TITLE
DTGB-159: Added tag styling without link support.

### PR DESCRIPTION
**Pull request by meirege**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.

### CODE REVIEW

**Review by Jeroen005 — APPROVED** [review]
Ervan uitgaande dat dit component is gevalideerd en de juiste benaming heeft.

File: styleguide/components/21-atoms/tag/_tag.scss [hunk 1]
```diff
@@ -1,28 +1,9 @@
...
-    &:visited {
-      @content;
-    }
-
-    &:hover,
-    &:focus {
-      cursor: pointer;
-    }
-  }
+  height: 1.5rem;
```
Comment (delrueba) [inline styleguide/components/21-atoms/tag/_tag.scss]: Is this needed?
Won't it break if your tag wraps over two lines?

Suggest using line-height.

Comment (delrueba) [reply styleguide/components/21-atoms/tag/_tag.scss]: Also: inline elements don't have a height property.

**Review by delrueba — CHANGES_REQUESTED** [review]

File: styleguide/components/21-atoms/tag/_tag.scss (hunk 1 above)
Comment (meirege) [reply styleguide/components/21-atoms/tag/_tag.scss]: Fixed it.

**Review by delrueba — APPROVED** [review]

**Pull request merged**